tests/outcomes/plot/test_pandas/test_drawings_data/test.py

```diff
@@ -0,0 +1,260 @@
+
+        if bar2.data['x'] != 'lab':
+            return wrong(f"Wrong 'x' value {bar2.data['x']}. Expected 'lab'")
+
+        if bar2.data['y'] != 'val':
+            return wrong(f"Wrong 'y' value {bar2.data['y']}. Expected 'val'")
+
+        # barh
+
+        barh1 = self.all_figures[4]
+        if barh1.type != 'barh':
+            return wrong(f'Wrong drawing type {barh1.type}. Expected barh')
+
+        if 'data' not in barh1.data or 'x' not in barh1.data or 'y' not in barh1.data:
+            return wrong(f"Expected 'data', 'x', 'y' key in the data dict of the barh drawing")
+
+        if type(barh1.data['data']) != pd.DataFrame:
+            return wrong(f"Wrong 'data' type {type(barh1.data['data'])}. Expected {pd.DataFrame}")
+
+        if barh1.data['x'] != 'lab':
+            return wrong(f"Wrong 'x' value {barh1.data['x']}. Expected 'lab'")
+
+        if barh1.data['y'] != 'val':
+            return wrong(f"Wrong 'y' value {barh1.data['y']}. Expected 'val'")
+
+        barh2 = self.all_figures[5]
+        if barh2.type != 'barh':
+            return wrong(f'Wrong drawing type {barh2.type}. Expected barh')
+
+        if 'data' not in barh2.data or 'x' not in barh2.data or 'y' not in barh2.data:
+            return wrong(f"Expected 'data', 'x', 'y' key in the data dict of the barh drawing")
+
+        if type(barh2.data['data']) != pd.DataFrame:
+            return wrong(f"Wrong 'data' type {type(barh2.data['data'])}. Expected {pd.DataFrame}")
+
+        if barh2.data['x'] != 'lab':
+            return wrong(f"Wrong 'x' value {barh2.data['x']}. Expected 'lab'")
+
+        if barh2.data['y'] != 'val':
+            return wrong(f"Wrong 'y' value {barh2.data['y']}. Expected 'val'")
+
+        # box
+
+        for box in (self.all_figures[6], self.all_figures[7]):
+            if box.type != 'box':
+                return wrong(f'Wrong drawing type {box.type}. Expected box')
+
+            if 'data' not in box.data or 'x' not in box.data or 'y' not in box.data:
+                return wrong(f"Expected 'data', 'x', 'y' key in the data dict of the box drawing")
+
+            if type(box.data['data']) != pd.DataFrame:
+                return wrong(f"Wrong 'data' type {type(box.data['data'])}. Expected {pd.DataFrame}")
+
+        # kde
+
+        for kde in (self.all_figures[8], self.all_figures[9]):
+            if kde.type != 'kde':
+                return wrong(f'Wrong drawing type {kde.type}. Expected kde')
+
+            if 'data' not in kde.data or 'x' not in kde.data or 'y' not in kde.data:
+                return wrong(f"Expected 'data', 'x', 'y' key in the data dict of the kde drawing")
+
+            if type(kde.data['data']) != pd.Series:
+                return wrong(f"Wrong 'data' type {type(kde.data['data'])}. Expected {pd.Series}")
+
+        # hexbin
+
+        for hexbin in (self.all_figures[10], self.all_figures[11]):
+            if hexbin.type != 'hexbin':
```

Comment:
It is also available in matplotlib and pandas
https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.hexbin.html
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.plot.hexbin.html